TestON/drivers/common/cli/onosdriver.py

```diff
@@ -173,7 +173,81 @@ def cleanInstall( self ):
                         'Runtime\sEnvironment\sto\scontinue',
                     'BUILD\sFAILURE',
                     'BUILD\sSUCCESS',
-                    'ONOS\$',
```

Comment:
So I think current master has some changes here that are conflicting with yours? You didn't have much changes in cleanInstall did you?